edb/api/errors.txt

```diff
@@ -111,8 +111,9 @@
 0x_05_02_00_03   MissingRequiredError
 
 0x_05_03_00_00   TransactionError
-0x_05_03_00_01   TransactionSerializationError  #SHOULD_RETRY
-0x_05_03_00_02   TransactionDeadlockError  #SHOULD_RETRY
+0x_05_03_01_00   TransactionConflictError
```

Comment:
I think you can specify `#SHOULD_RETRY` for the new `TransactionConflictError` (and there shouldn't be need to specify it for the leaf error types, right @tailhook?)

Reply:
updated

Reply:
@tailhook feel free to merge in the morning your time

Reply:
@1st1 were you expecting `edb gen-errors-json --client` to tag the deadlock and serialization errors with `SHOULD_RETRY`? It doesn't look like it does.

Reply:
Yeah, I did, that was my question to Paul.